packages/core/cache/src/types.js

```diff
@@ -12,7 +12,7 @@ export interface Cache {
   setBlob(key: string, contents: Buffer | string): Promise<void>;
   hasLargeBlob(key: string): Promise<boolean>;
   getLargeBlob(key: string): Promise<Buffer>;
-  setLargeBlob(key: string, contents: Buffer | string): Promise<void>;
```

Comment:
Not sure if this ever actually happens, but this is technically a breaking change for this package. Is it definitely required?

Reply:
It's not required but I figured it would be a nice clean up to do, since it's adding complexity. Is there a convention for tightening APIs like this?

I'll remove the API change anyway, since it's relatively easy to implement